sinon/lib/stub.py

```diff
@@ -26,12 +26,24 @@ def __init__(self, obj=None, prop=None, func=None):
         super(SinonStub, self).wrap2stub(self._stubfunc)
         self._copy = self._cond_args = self._cond_kwargs = self._oncall = None
         # Todo: target is a dirty hack
-        self._conditions = {"args":[], "kwargs":[], "action": [], "oncall": [], "target": self.obj}            
+        self._conditions = {"args":[], "kwargs":[], "action":[], "oncall":[], "target": self.obj}
+        self._conditions["default"] =  lambda *args, **kwargs: None
 
     def _append_condition(self, sinon_stub_condition, func):
```

Comment:
I didn't notice this last time, the parameters `sinon_stub_condition` and `func` weren't not used by this function. They could be removed.

Reply:
Hi Kir,

I don't understand. Both `sinon_stub_condition` and `func` are used inside of `_append_condition`:

```python
def _append_condition(self, sinon_stub_condition, func):
    self._conditions["args"].append(   sinon_stub_condition   ._cond_args)
    self._conditions["kwargs"].append(   sinon_stub_condition   ._cond_kwargs)
    self._conditions["oncall"].append(   sinon_stub_condition   ._oncall)
    self._conditions["action"].append(   func   )
    return self._conditions
```

Reply:
sorry, please ignore this comment, it's my misread.